bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaAdjustableSlatsRollerShutterHandler.java

```diff
@@ -87,7 +87,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             if (OnOffType.ON.equals(command)) {
                 sendCommand(COMMAND_SET_ROCKERPOSITION);
             }
-        } else {
+        } else if (CLOSURE_AND_ORIENTATION.equals(channelUID.getId())) {
+            String param = "[" + command.toString() + "]";
```

Comment:
Just for safety: What happens when the user sends "foobar" to this channel?

Reply:
The Tahoma cloud will return an error, that this command expects two parameters.
I had been thinking about doing a check for two numbers 0-100 separated by a comma, but decided to leave it on the Tahoma at the end...